packages/myst-parser/src/config.ts

```diff
@@ -69,4 +69,4 @@ export const MARKDOWN_IT_CONFIG = {
 };
 
 // List of valid TLDs to exclude from linkify
-export const EXCLUDE_TLDS = ['py', 'md'];
+export const EXCLUDE_TLDS = ['py', 'md', 'dot'];
```

Comment:
```suggestion
export const EXCLUDE_TLDS = ['py', 'md', 'dot', 'next'];
```

`.next` is another one from scipy content...